packages/bitcore-node/src/routes/api/address.ts

```diff
@@ -30,6 +31,66 @@ router.get('/:address', function(req, res) {
   ChainStateProvider.streamAddressUtxos(payload);
 });
 
+router.get('/:address/coins', async function (req, res) {
+  let { address, chain, network } = req.params;
+  // let { unspent, limit = 10, since } = req.query;
+  try {
+    let coins = await CoinStorage.collection.find({address, chain, network}).toArray();
```

Comment:
this could potentially be a very large amount of coins, for instance 1vaynert3x1kzbpzmgt2qdqrathirovi8 on BTC mainnet might blow this up

Reply:
Ok... then I suggest to set a limit perhaps find the last ```x``` amount of transactions.